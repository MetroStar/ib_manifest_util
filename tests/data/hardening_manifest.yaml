---
apiVersion: v1
name: opensource/metrostar/singleuser
tags:
  - singeluser_v12
args:
  BASE_IMAGE: 
  BASE_TAG: 
labels:
  # Name of the image
  org.opencontainers.image.title: 
  org.opencontainers.image.description: 
  org.opencontainers.image.licenses: 
  org.opencontainers.image.url: 
  org.opencontainers.image.vendor: 
  org.opencontainers.image.version: 
  mil.dso.ironbank.image.keywords: 
  mil.dso.ironbank.image.type: 
  mil.dso.ironbank.product.name: 
resources:
  - url: https://github.com/dirkcgrunwald/jupyter_codeserver_proxy-/archive/5596bc9c2fbd566180545fa242c659663755a427.tar.gz
    filename: code_server.tar.gz
    validation:
      type: sha256
      value: 7a286d6f201ae651368b65505cba7b0a137c81b2ac0fd637160d082bb14db032
  - url: https://conda.anaconda.org/conda-forge/linux-64/_openmp_mutex-4.5-2_gnu.tar.bz2
    filename: openmp_mutex-4.5-2_gnu.tar.bz2
    validation:
      type: sha256
      value: fbe2c5e56a653bebb982eda4876a9178aedfc2b545f25d0ce9c4c0b508253d22

maintainers:
  - email: demo@email.com
    name: Demo demo
    username: demo
    cht_member: false