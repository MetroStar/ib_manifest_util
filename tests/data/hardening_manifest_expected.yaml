---
apiVersion: v1
name: opensource/metrostar/singleuser
tags:
  - singeluser_v12
args:
  BASE_IMAGE:
  BASE_TAG:
labels:
  # Name of the image
  org.opencontainers.image.title:
  org.opencontainers.image.description:
  org.opencontainers.image.licenses:
  org.opencontainers.image.url:
  org.opencontainers.image.vendor:
  org.opencontainers.image.version:
  mil.dso.ironbank.image.keywords:
  mil.dso.ironbank.image.type:
  mil.dso.ironbank.product.name:
resources:
  - url: https://github.com/dirkcgrunwald/jupyter_codeserver_proxy-/archive/5596bc9c2fbd566180545fa242c659663755a427.tar.gz
    filename: code_server.tar.gz
    validation:
      type: sha256
      value: 7a286d6f201ae651368b65505cba7b0a137c81b2ac0fd637160d082bb14db032

maintainers:
  - email: jvelando@metrostarsystems.com
    name: Jonathan Velando
    username: jvelando
    cht_member: false
